cassiopeia/plotting/local.py

```diff
@@ -185,6 +185,7 @@ def create_indel_heatmap(
     indel_colors: Optional[pd.DataFrame] = None,
     indel_priors: Optional[pd.DataFrame] = None,
     random_state: Optional[np.random.RandomState] = None,
+    clustered_linprof: Optional[pd.DataFrame] = None,
```

Comment:
Remove? Isn't clustered_linprof is generated by 219? Or add to doc string if this argument is used.

Reply:
Ah yes, good catch. This is left over from some other code I was experimenting with.